### ISSUE TITLE
Make the smoke tests in TestReportStream run in parallel

### ISSUE THREAD

**Issue by jimduff-usds** [CDCgov/prime-reportstream#1334]
(fyi @tlosobo)
When you call `./prime test` with no args, it runs all the tests labelled as part of the Smoke Test.
Typically this is about 8 tests.
In addition, you can manually select a set of tests you want to run, for example:
`./prime test --run ping,repeatwaters,huge,toobig,intcontent`

Right now you have to wait a really long time, as they run sequentially.

This task is to experiment with running the set of tests all _in parallel._

You'll need to launch each in a separate thread, then the tricky part is you have to do a join() to wait for them all to finish.  Look in the code for the `repeatwaters` test to see an example of this.

If they don't work well running _all_ in parallel, then you could try running two or four in parallel at a time.

Also make sure a manually selected set of tests runs well in parallel, as in the above example.
As an ultimate test, you can try running *all* the tests!
(Use `./prime test --list` to see a list of them all)
Then you'll need to ask Jim or someone to test what happens when you run them against our Staging system.

**Comment by carlosfelix2** [comment]
Already implemented as part of another issue
